nautobot_ssot_infoblox/diffsync/adapters/nautobot.py

```diff
@@ -177,7 +213,10 @@ def load_vlangroups(self):
     def load_vlans(self):
         """Load VLANs from Nautobot."""
         default_cfs = get_default_custom_fields(cf_contenttype=ContentType.objects.get_for_model(VLAN))
-        for vlan in VLAN.objects.all():
+        for vlan in VLAN.objects.filter(group__isnull=False):
```

Comment:
why this constraint?

Reply:
This is to match Infoblox's way of providing their VLANs. In Infoblox, all VLANs must be associated with a VLAN View (VLAN Group). This should ensure that we're only dealing with data that was pulled from Infoblox before.

Reply:
maybe add a comment to explain it? (to avoid same question again)